doc/guide-startmany.md

```diff
@@ -0,0 +1,153 @@
+#start-many Setup Guide
+
+## Two Options for Setting up your Wallet
+There are many ways to setup a wallet to support start-many. This guide will walk through two of them.
+
+1. [Importing an existing wallet (recommended if you are consolidating wallets).](#option1)
+2. [Sending 1,000 DRK to new wallet addresses.](#option2)
+
+## <a name="option1"></a>Option 1. Importing an existing wallet
+
+This is the way to go if you are consolidating multiple wallets into one that supports start-many. 
+
+### From your single-instance MasterNode Wallet
+
+Open your QT Wallet and go to console (from the menu select Tools => Debug Console)
+
+Dump the private key from your MasterNode's pulic key.
+
+```
+walletpassphrase [your_wallet_passphrase] 600
+dumpprivkey [mn_public_key]
+```
+
+Copy the resulting priviate key. You'll use it in the next step.
+
+### From your multi-instance MasterNode Wallet
+
+Open your QT Wallet and go to console (from the menu select Tools => Debug Console)
+
+Import the private key from the step above.
+
+```
+walletpassphrase [your_wallet_passphrase] 600
+importprivkey [single_instance_private_key]
+```
+
+The wallet will re-scan and you will see your available balance increase by the amount that was in the imported wallet.
+
+[Skip Option 2. and go to Create masternode.conf file](#masternodeconf)
+
+## <a name="option2"></a>Option 2. Starting with a new wallet
+
+[If you used Option 1 above, then you can skip down to Create masternode.conf file.](#masternodeconf)
+
+### Create New Wallet Addresses
+
+1. Open the QT Wallet.
+2. Click the Receive tab.
+3. Fill in the form to request a payment.
+    * Label: mn01
+    * Amount: 1000 (optional)
+    * Click *Request payment*
+5. Click the *Copy Address* button
+
+Create a new wallet address for each MasterNode.
+
+Close your QT Wallet.
+
+### Send 1,000 DRK to New Addresses
+
+Just like setting up a standard MN. Send exactly 1,000 DRK to each new address created above.
+
+### Create New Masternode Private Keys
+
+Open your QT Wallet and go to console (from the menu select Tools => Debug Console)
+
+Issue the following:
```

Comment:
Good find @freynder ! Having the same masternode private key for many masternodes does not break security in any way but if someone break into one of your masternodes he can stop all of your masternodes using the same masternode private key which can be an issue. So, yes, it's better to have a separate masternode private key for each masternode and worth mentioning for sure.